rust/index/src/fulltext/types.rs

```diff
@@ -161,6 +162,40 @@ impl FullTextIndexWriter {
         Ok(())
     }
 
+    pub async fn prefetch_before_writing_to_blockfiles(
```

Comment:
discussed offline that it's probably better for the orchestrator to pass in the keys to prefetch that way in future if it has to control the size of work then it can do so

Reply:
e0f7533da